model/botKey.go

```diff
@@ -30,10 +30,10 @@ func (b BotKey) String() string {
 }
 
 // MakeSortedBotKey makes a BotKey by sorting the passed in assets
-func MakeSortedBotKey(assetA horizon.Asset, assetB horizon.Asset) *BotKey {
+func MakeSortedBotKey(assetA horizon.Asset, assetB horizon.Asset) (*BotKey, error) {
 	var assetBaseCode, assetBaseIssuer, assetQuoteCode, assetQuoteIssuer string
 	if assetA.Type == native && assetB.Type == native {
-		log.Fatal("invalid asset types, both cannot be native")
+		return nil, fmt.Errorf("invalid asset types, both cannot be native")
```

Comment:
let's not fix any place where we are using log.Fatal right now. this PR should be a simple translation of old format to new.

Reply:
ok